appengine/genetics/style.css

```diff
@@ -0,0 +1,171 @@
+#blockly {
+    position: fixed;
+    bottom: 10px;
+}
+
+#editor {
+    font-size: 16pt;
+    position: fixed !important;
+    height: auto !important;
+    bottom: 10px;
+    border: 1px solid #ddd;
+}
+
+#visualization {
+    position: relative;
+}
+
+#display {
+    border: 1px solid #ddd;
+    vertical-align: bottom;
+}
+
+#geneticsStatTable {
+    width: 406px;
+    max-width: 406px;
+    margin-left: -2px;
+    height: 20px;
+}
+
+#geneticsStatTable>tr>td {
+    padding: 0;
+    border: 1px solid;
+    background: #ccc;
+    border-radius: 2px;
+}
+
+#tabarea {
+    position: fixed !important;
+}
+
+/*
```

Comment:
Is there some way to import the closure stuff, rather than including it manually here? I feel that, at least, we might want to put it in a separate file and have instructions somewhere for how to generate it.